tests/acceptance/features/webUIPublicShareLink/publicShareLinkUppercaseLetters.feature

```diff
@@ -1,27 +1,35 @@
 @webUI
-Feature: enforce password_policy on publicly shared links
+Feature: enforce the required number of uppercase letters in a password on the when creating a public share link
 
-	As an admin
-	I want to enforce password_policy on publicly shared links
-	So that publicly shared links are protected from un-authorized access
+	As an administrator
+	I want public link share to always contain a required number of uppercase letters
+	So that users cannot set passwords that are too easy to guess
 
 	Background:
 		Given these users have been created:
 			|username|password|displayname|email       |
-			|user1   |ABC123  |User One   |u1@oc.com.np|
+			|user1   |abc123  |User One   |u1@oc.com.np|
```

Comment:
This works, but it might be a bit dangerous because ``abc123`` does not meet the new password policy set below.

Reply:
This should work fine because we are changing the `password_policy` after the user has been created.